schemas/browserlib/extract-dfns.json

```diff
@@ -63,6 +63,10 @@
       },
       "definedIn": {
         "type": "string"
+      },
+      "prose": {
```

Comment:
should we name this htmlProse or something of the sort to make it clear this is markup, not plain text?